zorg/jenkins/jobs/jobs/clang-stage2-cmake-RgSan

```diff
@@ -1,6 +1,8 @@
 pipeline {
     options {
         disableConcurrentBuilds()
+
+        timeout(time: 12, unit: 'HOURS')
```

Comment:
Not sure if 12 hours is enough, looks like the last passing build took 15 hours: https://green.lab.llvm.org/job/llvm.org/job/clang-stage2-cmake-RgSan/535/

Maybe we need a faster machine for the sanitizer job :)

Reply:
good point, for now I'll bump it to 24